pkg/kubectl/cmd/version.go

```diff
@@ -17,17 +17,25 @@ limitations under the License.
 package cmd
 
 import (
+	"errors"
 	"fmt"
-	"io"
-
```

Comment:
@eparis yeah probably got deleted and re-added after some change, ill go ahead and make the format change.